x/evm/statedb/testdata/flash_loan.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package testdata
+
+import (
+	_ "embed" // embed compiled smart contract
+	"encoding/json"
+	"errors"
+
+	evmtypes "github.com/evmos/evmos/v16/x/evm/types"
+)
+
+//go:embed FlashLoan.json
+var FlashLoanJSON []byte //nolint:golint // used to embed the compiled contract
+
+func LoadFlashLoanContract() (evmtypes.CompiledContract, error) {
```

Comment:
nit: we have other function with the same name that are just loading. Here we load and compile
```suggestion
func LoadAndCompileFlashLoanContract() (evmtypes.CompiledContract, error) {
```

Reply:
no, that's not the case - the files are already compiled, we are just changing the format from the Hardhat format to the format, that is used in the repo